assigner.py

```diff
@@ -23,8 +24,8 @@ def main():
     parser.add_argument('-d', '--debug', action="store_true", help='Enable debug mode')
     parser.add_argument('--dry-run', action="store_true", help='Perform all read operations, and no write operations. This means no bug '
             'will be updated, CASA won\'t be updated, etc.')
-    parser.add_argument('--rra', action="store_true", help='Assign RRAs automatically')
-    parser.add_argument('--casa', action="store_true", help='Find resolved bugs which status needs to be updated in CASA/Biztera')
+    parser.add_argument('--configfile', action="store_true", help='Config file that specifies all the parameters we need to assign bugs')
```

Comment:
```
action="store_true",
```
is incorrect in this case because it takes a param

Reply:
to be clear, in case you havent see the arg parser before: just remove that argument entierely so that it works as a param
store_true means it will store the boolean value true in "args.configfile" instead of the path you're passing

Reply:
gotcha